openghg/objectstore/metastore/_metastore.py

```diff
@@ -256,5 +308,5 @@ def delete(self, metadata: MetaData, delete_one: bool = True) -> None:
                 be deleted.
```

Comment:
same as above

Reply:
I just updated the local variable names here, so I'm not sure if the docstring needs to change. (Actually... I didn't intend to change `query` to `_query` everywhere... my editor's "rename variable" function changed more than I wanted it to...)